Project.toml

```diff
@@ -18,16 +17,28 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
 [compat]
+ColorVectorSpace = "0.6, 0.7, 0.8"
+DataStructures = "0.17, 0.18"
+ImageCore = "0.8"
+ImageFiltering = "0.6"
+Interpolations = "0.10, 0.11, 0.12"
+MappedArrays = "0.2, 0.3"
```

Comment:
FYI, it is doable to drop the direct dependency on `MappedArrays` so that we don't need to maintain the compatibility.

See https://github.com/JuliaImages/ImageFiltering.jl/pull/192 for how it's done.

Reply:
Thanks for the excellent suggestion.